teams/teams-ps/teams/Set-CsTeamsAIPolicy.md

```diff
@@ -121,6 +121,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Description
```

Comment:
What are the possible values for this? Are there any limitations?

Reply:
Should be any string for possible values - this is commonly used for all policies, you can search for "### -Description" in the document repo